src/implementations/twig/components/gallery/__snapshots__/gallery.test.js.snap

```diff
@@ -3138,7 +3138,7 @@ exports[`Gallery Default renders correctly with old data 1`] = `
             </div>
             <video
               aria-label="EU law - Video"
-              class="ecl-gallery__image"
+              class="ecl-video ecl-gallery__image"
               poster="https://inno-ecl.s3.amazonaws.com/media/examples/example-image5.jpg"
```

Comment:
For some reason we lost the change in the gallery data, using another html video.
It could be good to bring it back while we are changing the gallery and also use it in the video component data)
https://github.com/ec-europa/europa-component-library/pull/3554/files#diff-330b93090781d35d9db384dca62ddc8ab9726a69caccb37617070e162ff45beaR92